### DISCUSSION TITLE
DWIN Gotcha missing

### DISCUSSION THREAD

**Discussion by m3talp4nda** [Jyers/Marlin#1577] [Q&A]
I downloaded the source zip file and decompressed it, but I cannot find the DWIN files anywhere inside it. Am I just blind, or is it missing from the from zip file, and if it is missing, how do I download the DWIN gotcha files to update my E3V2 screen?

**Comment by mostlyflawles** [comment]
Yes, I also noticed they were missing in the release file. But if you download the whole "Code" project the files are there in the zip.

**Reply by m3talp4nda** [reply]
I'll have to try that later.
